docs/content/en/docs/clustermgmt/cluster-backup-restore/restore-cluster.md

```diff
@@ -0,0 +1,148 @@
+
+If the management cluster is still accessible through the API server, and the underlying infrastructure machines are not changed after the etcd backup was taken, simply follow the [External etcd backup and restore]({{< relref "../etcd-backup-restore/etcdbackup" >}}) to restore the management cluster itself from the backup.
+
+### Cluster not accessible or infrastructure components changed after etcd backup was taken
+
+If the cluster is no longer accessible in any means, or the infrastructure machines are changed after the etcd backup was taken, restoring this management cluster itself from the outdated etcd backup will not work. Instead, you need to create a new management cluster, and migrate all the EKS Anywhere resources of the old workload clusters to the new one, so that the new management cluster can maintain the new ownership of managing the existing workload clusters. Below is an example of migrating a failed management cluster `mgmt-old` with its workload clusters `w01` and `w02` to a new management cluster `mgmt-new`:
+
+1. Create a new management cluster to which you will be migrating your workload clusters later.
+
+    You can define a cluster config similar to your old management cluster, with a different cluster name, and run cluster creation of the new management cluster.
+
+    ```sh
+    eksctl anywhere create cluster -f mgmt-new.yaml
+    ```
+
+1. Move the custom resources of all the workload clusters to the new management cluster created above.
+
+    Using vSphere provider as an example, we are moving the Cluster API custom resources, such as `vpsherevms`, `vspheremachines` and `machines` of the **workload clusters**, from the old management cluster to the new management cluster created in above step. Using the `--filter-cluster` flag with `clusterctl move` command so that we are only targeting the custom resources from the workload clusters.
+
+
+    ```bash
+    MGMT_CLUSTER_OLD="mgmt-old"
+    MGMT_CLUSTER_NEW="mgmt-new"
+    MGMT_CLUSTER_NEW_KUBECONFIG=${MGMT_CLUSTER_NEW}/${MGMT_CLUSTER_NEW}-eks-a-cluster.kubeconfig
+    WORKLOAD_CLUSTER_1="w01"
+    WORKLOAD_CLUSTER_2="w02"
+
+    # Substitute the workspace path with the workspace you are using
+    WORKSPACE_PATH="/home/ec2-user/eks-a"
+    CLUSTER_STATE_BACKUP_LATEST=$(ls -Art ${WORKSPACE_PATH}/${MGMT_CLUSTER_OLD} | grep 'cluster-state-backup' | tail -1)
+    CLUSTER_STATE_BACKUP_LATEST_PATH=${WORKSPACE_PATH}/${MGMT_CLUSTER_OLD}/${CLUSTER_STATE_BACKUP_LATEST}/
+
+    # Substitute the container version with whatever EKS Anywhere CLI version you are using
+    CONTAINER=public.ecr.aws/eks-anywhere/cli-tools:v0.16.2-eks-a-41
+
+
+    # The clusterctl move command needs to be executed on each workload cluster you have. It will will only move the workload cluster resources from the EKS Anywhere backup to the new management cluster. If you have multiple workload clusters, you have to run the command for each cluster as shown below.
```

Comment:
nit: can we break this up into multiple lines so it's easier to read when rendererd?
```suggestion
    # The clusterctl move command needs to be executed for each workload cluster. 
    # It will only move the workload cluster resources from the EKS Anywhere backup to the new management cluster. 
    # If you have multiple workload clusters, you have to run the command for each cluster as shown below.
```